back/lib/locale.rb

```diff
@@ -0,0 +1,19 @@
+class Locale
```

Comment:
I think it would be useful to add more methods to this class at some point, but considered this out of scope for this PR.